xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -77,10 +77,10 @@ struct DemuxPacket;
 #define PVR_STREAM_MAX_STREAMS 20
 
 /* current PVR API version */
-#define XBMC_PVR_API_VERSION "5.1.1"
+#define XBMC_PVR_API_VERSION "6.0.0"
```

Comment:
I suggested this, but I think I was wrong. Minor bump should be the right thing to do for new api functions, that are actually called by Kodi, right?